vultr-dns.py

```diff
@@ -1,43 +1,49 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import requests
 import sys
 import os
-import string
 from time import sleep
 
 # Configure here
 VULTR_API_KEY = "put your api key here"
-VULTR_BIND_DELAY = 30
+VULTR_BIND_DELAY = 60
 
 
-def vultr_request(method, path, data=None):
-    url = "https://api.vultr.com/v1{}".format(path)
+def vultr_request(method, path, data=None, json=None):
+    url = "https://api.vultr.com/v2{}".format(path)
 
-    resp = requests.request(method, url, data=data, headers={
-                            "API-Key": VULTR_API_KEY})
+    resp = requests.request(
+        method,
+        url,
+        data=data,
+        json=json,
+        headers={"Authorization": "Bearer " + VULTR_API_KEY},
+    )
     resp.raise_for_status()
-    if resp.headers['Content-Type'] == 'application/json':
+    if resp.headers["Content-Type"] == "application/json":
         return resp.json()
     return resp.text
```

Comment:
`data` and `json` appear to be redundant. Let's keep only one of them.

Reply:
Okay, so let's just leave the `json`. As [requests doc](https://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests) says, using json parameter will change the `Content-Type` in the header to `application/json`. That's Vultr API v2 needs.